Sources/MapboxMaps/Foundation/Extensions/Core/CoordinateBounds.swift

```diff
@@ -5,27 +5,106 @@ import MapboxCoreMaps
 
 public extension CoordinateBounds {
 
-    /// The northwest coordinate of an internal `CoordinateBounds` type.
+    /// Returns the southern latitude of the bounds.
+    var south: CLLocationDegrees { __south() }
+
+    /// Returns the western longitude of the bounds.
+    var west: CLLocationDegrees { __west() }
+
+    /// Returns the northern latitude of the bounds.
+    var north: CLLocationDegrees { __north() }
+
+    /// Returns the eastern longitude of the bounds.
+    var east: CLLocationDegrees { __east() }
+
+    /// The centerpoint of this `CoordinateBounds` calculated by simple interpolation.
+    /// This is a non-geodesic calculation which is not the geographic center.
+    var center: CLLocationCoordinate2D {
+        return __center()
+    }
+
+    /// Returns whether the bounds are empty or not.
+    var isEmpty: Bool { __isEmpty() }
+
+    /// The absolute distance, in degrees, between the north and south boundaries of these bounds.
+    ///
+    /// One degree of latitude is approximately 111 kilometers (69 miles).
+    var latitudeSpan: CLLocationDegrees {
+        __latitudeSpan()
+    }
```

Comment:
inconsistent formatting. please be consistent about whether to include `return` and whether to wrap or write all on one line